YouTube API Task/getDataFromAPI.js

```diff
@@ -0,0 +1,13 @@
+function getData() {
+
+    var searchTerm = document.querySelector("input#takeSearchInput") ? document.querySelector("input#takeSearchInput").value || "trailers" : "trailers";
```

Comment:
I will rename the variables.
"Also if the user doesn't enter any value in the search box, then you are taking trailers as the default text and performing a search. But this approach isn't correct. "
I did this to provide user with the home page, otherwise user will get a blank screen.